File: medium/1980/main.c
char* findDifferentBinaryString(char** nums, int numsSize) {
  int len = strlen(nums[0]);
  char *res = (char*) malloc(len * sizeof(char)+1);
  res[len] = '\0';

  for (int i = 0; i < numsSize; i++) {
    switch (nums[i][i]) {
      case '0':
        res[i] = '1';
        break;
      default:
        res[i] = '0';
        break;
    }
  }

  return res;
    
}

File: medium/1980/main.rs
impl Solution {
    pub fn find_different_binary_string(nums: Vec<String>) -> String {
        let mut res = String::new();

        for (i, s) in nums.iter().enumerate() {
            let c = if s.as_bytes()[i] == b'0' {'1'} else {'0'};
            res.push(c);
        }

        res
        
    }
}
